index.html

```diff
@@ -3458,7 +3458,7 @@ <h2>A project for first-time contributions</h2>
 
             <!-- ________ Contributor's card END ________  -->
         <!-- ________ Shubhankar's card START ________  -->
-        <div class="card">
+        <!--<div class="card"></div>-->
```

Comment:
You should completely discard the changes here and leave it as it was before
This creates a problem in this card and breaks the layout
![image](https://user-images.githubusercontent.com/29199184/56473769-156b9e80-6470-11e9-8f05-4b4a6ebd6bd0.png)